packages/ui-logic/src/time.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { displayPeriod } from "./time";
+
+describe("displayPeriod", () => {
+  it("correctly displays seconds", () => {
+    expect(displayPeriod(1)).toBe("1s");
+    expect(displayPeriod(1)).not.toBe("0d 0h 0m 1s");
```

Comment:
All those ".not.toBe" tests can be removed. If you know what string it is, then it is not anything else. So the extra tests are not adding value and are hard to maintain.